cookbook/email/email.rst

```diff
@@ -123,7 +123,21 @@ an email is pretty straightforward::
     }
 
 To keep things decoupled, the email body has been stored in a template and
-rendered with the ``renderView()`` method.
+rendered with the ``renderView()`` method. The ``registration.html.twig``
+template might look something like this:
+
+.. code-block:: html+jinja
+
+    {# app/Resources/views/Emails/registration.html.twig #}
+    <h3>You did it! You registered!</h3>
+
+    {# example, assuming you have a route named "login" $}
+    To login, go to: {{ url('login') }}.
+
+    Thanks!
+
+    {# Makes an absolute URL to the /images/logo.png file #}
+    <img src="{{ absolute_url(asset('images/logo.png')) }}"
```

Comment:
You're right. I'll open an issue there (imo, 2.7 should still support PHP templating, no matter what will happen in 3.x)